lib/auth/password_test.go

```diff
@@ -307,42 +324,74 @@ func TestServer_ChangePassword_FailsWithoutOldPassword(t *testing.T) {
 	server := newTestTLSServer(t)
 	mfa := configureForMFA(t, server)
 	authServer := server.Auth()
-	ctx := context.Background()
-
+	ctx := authz.ContextWithUser(
+		context.Background(),
+		authz.LocalUser{Username: mfa.User, Identity: tlsca.Identity{Username: mfa.User}},
+	)
 	username := mfa.User
-	newPass := []byte("capybarasarecool123")
 
-	userClient, err := server.NewClient(TestUser(username))
-	require.NoError(t, err)
-	defer userClient.Close()
-
-	// Acquire and solve an MFA challenge.
-	mfaChallenge, err := userClient.CreateAuthenticateChallenge(ctx, &proto.CreateAuthenticateChallengeRequest{
-		Request: &proto.CreateAuthenticateChallengeRequest_ContextUser{
-			ContextUser: &proto.ContextUser{},
+	tests := []struct {
+		name                        string
+		userVerificationRequirement string
```

Comment:
Drop this and let the string be empty in the request? It should be more reflective of production usage.

```suggestion
```